src/cosmos/base/query/v1beta1/pagination.ts

```diff
@@ -1,17 +1,15 @@
-/* eslint-disable */
-import Long from "long";
 import * as _m0 from "protobufjs/minimal";
-
+import { Long, isSet, bytesFromBase64, base64FromBytes, DeepPartial, Exact } from "@osmonauts/helpers";
 export const protobufPackage = "cosmos.base.query.v1beta1";
 
 /**
  * PageRequest is to be embedded in gRPC request messages for efficient
  * pagination. Ex:
  *
- *  message SomeRequest {
- *          Foo some_parameter = 1;
- *          PageRequest pagination = 2;
- *  }
+ * message SomeRequest {
+ * Foo some_parameter = 1;
+ * PageRequest pagination = 2;
+ * }
```

Comment:
Huh, what happened to this comment?

Reply:
something in the protobuf.js — I can take a deeper look and see if I can fork and fix this. Currently this is the source data. Potentially `protoc` uses a different codepath to protobuf.js which is why this is new. This is definitely not in telescope though we can potentially fix since we already have a fork of protobuf.js to handle issues like this